pages/Ethereum/Identity.md

```diff
@@ -51,7 +31,7 @@ While at least one owner is required, additional public keys may be considered t
 
 ## Delegation
 
-Delegation allows adding public keys that are allowed to sign announcements or perform other actions in addition to the "owner" public key.
+Delegation allows adding additional public keys that are allowed to sign announcements or perform other actions in addition to the owner public key(s).
```

Comment:
"Delegation allows adding additional public keys (additional to the owner public key, that is) that are allowed to perform certain actions on behalf of the owner."  ?

I'm partly offering the rewrite to test my own understanding of what you wrote, but also (if my understanding was correct) to offer a clearer version.  Delegation is a precise thing; we need to make clear both exactly what it is and what its general role in DSNP is.  The latter concern is what motivated my "on behalf of" (as an alternative to "in addition to".  Although if "on behalf of" is not quite the right phrase then let's find something else.  One way or another, I think the spec needs to say something about what delegation *means* -- i.e., what someone who sees a delegated announcement can conclude about the upstream/original owning public key.

Reply:
That is correct. Here is an expanded version from your suggestion:

```suggestion
Delegation allows adding additional public keys, in addition to the owner public key(s).
These delegated keys are allowed to perform certain actions on behalf of the owner based on the delegate's "role".```